### DISCUSSION TITLE
livewire 2.10.8 not working for me

### DISCUSSION THREAD

**Discussion by mbagusrd** [livewire/livewire#5466] [7. Bugs (archived)] [answered]
Hi, I'm using apache2 with php 8.2 and laravel 8, before upgrade to v2.10.8, my livewire components working successfully. But after upgrade to v2.10.8, I got error 404. Is it bug or something I must change in my components?

![image](https://user-images.githubusercontent.com/19277896/209509754-df321c8d-e0df-426e-b973-f3e7a0040552.png)

**Comment by lam0819** [comment]
Could you check, is it related to  https://github.com/livewire/livewire/pull/4570

**Comment by joshhanley** [comment, marked as answer]
@mbagusrd there was a bug released in v2.10.8. I have submitted a PR #5490 with a fix. Hope this helps!

**Comment by mbagusrd** [comment]
Thank you so much for your help

**Reply by joshhanley** [reply]
@mbagusrd no worries! 🙂
